explorer/urlMigrations/CO2UrlMigration.ts

```diff
@@ -39,16 +38,10 @@ export const co2UrlMigration: UrlMigration = (url: Url) => {
     const explorerSlug = getExplorerSlugFromUrl(url)
     if (explorerSlug !== EXPLORER_SLUG) return url
 
-    // if there is no patch param, then it's an old URL
```

Comment:
Yeah I was wondering that too. The migrations are idempotent (I think?) so they are safe to run repeatedly.

But that little uncertainty is an issue, and for the future too. If we introduce a non-idempotent migration at some point and it goes to production, it might be a bit painful to fix if it goes unnoticed for a while.

So possibly, we can introduce a `version` param early, and eagerly increment it whenever we have a new migration. But in this case I thought it's safe not to do that.

What do you think?

Reply:
Right. Yes, if the migration really is idempotent then we probably don't need it right at this point.

On the other hand, having a version param already might make URL detection and migration a lot easier going forward, so it might be worth it to introduce it already?

Reply:
Yeah, let's move this to a separate Notion issue: [Introduce version param in explorer URLs?](https://www.notion.so/Introduce-version-param-in-explorer-URLs-8d3695f14f144d1aa6c5dfcd6ca19fa0)